src/MRA.Jobs.Client/Services/FileService/FileService.cs

```diff
@@ -0,0 +1,43 @@
+using System.Net;
+using System.Net.Http.Headers;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Components.Forms;
+using MudBlazor;
+
+namespace MRA.Jobs.Client.Services.FileService;
+
+public class FileService : IFileService
+{
+    private readonly HttpClient _client;
+    private readonly ISnackbar _snackbar;
+
+    public FileService(HttpClient client, ISnackbar snackbar)
+    {
+        _client = client;
+        _snackbar = snackbar;
+    }
+
+    public async Task<string> UploadAsync(IBrowserFile file)
+    {
+        var fileContent = new StreamContent(file.OpenReadStream()); //todo check file size;
+        fileContent.Headers.ContentType = new MediaTypeHeaderValue(file.ContentType);
+        var content = new MultipartFormDataContent();
+        content.Add(fileContent, "\"file\"", file.Name);
+
+        var fileUploadResponse = await _client.PostAsync("/api/File/upload", content);
+        switch (fileUploadResponse.StatusCode)
+        {
+            case HttpStatusCode.OK:
+                _snackbar.Add("CV upload success", Severity.Success);
+                return (await fileUploadResponse.Content.ReadFromJsonAsync<FileUploadResponse>()).Key;
+            default:
+                _snackbar.Add("CV upload error", Severity.Error);
+                return "";
+        }
```

Comment:
Code is not releasing resources somewhere. I am getting an error : 
{"type":"https://tools.ietf.org/html/rfc7231#section-6.6.1","title":"An error occurred while processing your request.","status":500,"detail":"The process cannot access the file 'C:\\Users\\dilshodk\\Documents\\MraFiles\\5b9e6e3e-37ae-4e80-b77f-280add1985fb_Nuclues.pdf' because it is being used by another process.","traceId":"0HMV9K2A118SQ:0000000D","stackTrace":"   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)\r\n   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)\r\n   at System.IO.File.OpenHandle(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)\r\n   at System.IO.File.ReadAllBytesAsync(String path, CancellationToken cancellationToken)\r\n   at MRA.Jobs.Infrastructure.Services.FileService.Download(String key) in C:\\Repos\\MRA-Jobs\\src\\MRA.Jobs.Api\\Infrastructure\\Services\\FileService.cs:line 41\r\n   at MRA.Jobs.Web.Controllers.ApplicationsController.DownloadCv(String key) in C:\\Repos\\MRA-Jobs\\src\\MRA.Jobs.Api\\WebUI\\Controllers\\ApplicationsController.cs:line 34\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|26_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)"}